src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.spring.validation;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<Password, UserRequestDto> {
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        String password = userRequestDto.getPassword();
+        return Objects.equals(password, userRequestDto.getRepeatPassword());
```

Comment:
Do you really need a `password` variable? 
```suggestion
        return Objects.equals(userRequestDto.getPassword(), userRequestDto.getRepeatPassword());
```

Reply:
sorry, I was correcting this method and forgot about the variable)